wgpu-types/src/lib.rs

```diff
@@ -638,6 +638,17 @@ pub struct SwapChainDescriptor {
     pub present_mode: PresentMode,
 }
 
+#[repr(C)]
+#[derive(Debug)]
+pub enum SwapChainStatus {
```

Comment:
We only put stuff in wgpu-types that A) needs to be user-visible, and B) needs to be shared between the Web and the native targets. I don't think this status enum fulfills these criteria.

Reply:
Yeah, originally I put it in `wgpu-core`, but I moved it to `wgpu-types` because of point B, since otherwise I couldn't use it in the signature of `wgpu-rs`'s, `Context::swap_chain_get_next_texture` when building for the web. Maybe there's a better way to handle that?